tools/stern/src/Stern/Intra.hs

```diff
@@ -832,8 +832,8 @@ getUserNotifications uid = do
     fetchAll xs start = do
       userNotificationList <- fetchBatch start
       let batch = view queuedNotifications userNotificationList
-      if (not . null) batch && (view queuedHasMore userNotificationList)
-        then fetchAll (batch ++ xs) (Just . (view queuedNotificationId) $ last batch)
+      if (not . null) batch && view queuedHasMore userNotificationList
```

Comment:
I'm always surprised that there is no alternative to `not . null`

Reply:
Indeed. Also sounds like that's a hard naming problem. Any ideas?

Reply:
`isJust . nonEmpty` ? ;)

`isJust . uncons` ?

Reply:
How about `isNotEmpty = not . null`?